packages/mobx-devtools-mst/package.json

```
@@ -23,7 +23,7 @@
     "build": "cross-env NODE_ENV=production webpack"
   },
   "peerDependencies": {
-    "mobx": "^2.2.0 || ^3.0.0 || ^4.0.0 || ^5.0.0",
+    "mobx": "^2.2.0 || ^3.0.0 || ^4.0.0 || ^5.0.0 || ^6.0.0 || ^6.13.0",
```

```suggestion
    "mobx": "^2.2.0 || ^3.0.0 || ^4.0.0 || ^5.0.0 || ^6.0.0",
```

I think ^6.0.0 should work to do what ^6.13.0 is doing here. Was there a specific reason you had wanted to include that?